cmd/fyne_demo/screens/container.go

```diff
@@ -0,0 +1,109 @@
+package screens
+
+import (
+	"fmt"
+	"image/color"
+
+	"fyne.io/fyne"
+	"fyne.io/fyne/canvas"
+	"fyne.io/fyne/layout"
+	"fyne.io/fyne/theme"
+	"fyne.io/fyne/widget"
+)
+
+// ContainerScreen loads a tab panel for containers and layouts
+func ContainerScreen() fyne.CanvasObject {
+	return widget.NewTabContainer(
+		widget.NewTabItem("Split", makeSplitTab()),
+		widget.NewTabItem("Scroll", makeScrollTab()),
+		// layouts
+		widget.NewTabItem("Border", makeBorderLayout()),
+		widget.NewTabItem("Box", makeBoxLayout()),
+		widget.NewTabItem("Grid", makeGridLayout()),
+	)
+}
+
+func makeBorderLayout() *fyne.Container {
+	top := makeCell()
+	bottom := makeCell()
+	left := makeCell()
+	right := makeCell()
+	middle := widget.NewLabelWithStyle("BorderLayout", fyne.TextAlignCenter, fyne.TextStyle{})
+
+	borderLayout := layout.NewBorderLayout(top, bottom, left, right)
+	return fyne.NewContainerWithLayout(borderLayout,
+		top, bottom, left, right, middle)
+}
+
+func makeBoxLayout() *fyne.Container {
+	top := makeCell()
+	bottom := makeCell()
+	middle := widget.NewLabel("BoxLayout")
+	center := makeCell()
+	right := makeCell()
+
+	col := fyne.NewContainerWithLayout(layout.NewVBoxLayout(),
+		top, middle, bottom)
+
+	return fyne.NewContainerWithLayout(layout.NewHBoxLayout(),
+		col, center, right)
+}
+
+func makeCell() fyne.CanvasObject {
+	rect := canvas.NewRectangle(&color.RGBA{128, 128, 128, 255})
+	rect.SetMinSize(fyne.NewSize(30, 30))
+	return rect
+}
+
+func makeGridLayout() *fyne.Container {
+	box1 := makeCell()
+	box2 := widget.NewLabel("Grid")
+	box3 := makeCell()
+	box4 := makeCell()
+
+	return fyne.NewContainerWithLayout(layout.NewGridLayout(2),
+		box1, box2, box3, box4)
+}
+
+func makeScrollTab() fyne.CanvasObject {
+	list := widget.NewHBox()
+	list2 := widget.NewVBox()
+
+	for i := 1; i <= 20; i++ {
```

Comment:
On my external display 20 is not enough buttons to reach the bottom, can we increase this?

Reply:
Maybe with a slider to define it or something like that.

Reply:
I don't understand - add a slider to the scroll demo to define how many items are inside a container?

Reply:
This would be a nice demo for bindings, not sure it is necessary here